<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmBuscarApunteContable</class>
 <widget class="QDialog" name="frmBuscarApunteContable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="1">
    <widget class="QPushButton" name="btnCancelar">
     <property name="text">
      <string>Cancelar</string>
     </property>
     <property name="icon">
      <iconset resource="../maya.qrc">
       <normaloff>:/Icons/PNG/close.png</normaloff>:/Icons/PNG/close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnAceptar">
     <property name="text">
      <string>Aceptar</string>
     </property>
     <property name="icon">
      <iconset resource="../maya.qrc">
       <normaloff>:/Icons/PNG/OK.png</normaloff>:/Icons/PNG/OK.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTableView" name="tablaapuntes"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5">
       <widget class="QDateEdit" name="dateFecha_fin">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="txtImporte_fin">
        <property name="text">
         <string>99.999.999,99</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>169</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="5" colspan="2">
       <widget class="QLineEdit" name="txtdesc_subcta"/>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="btnBuscar_subcta">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../maya.qrc">
          <normaloff>:/Icons/PNG/search2.png</normaloff>:/Icons/PNG/search2.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>De la subcuenta:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtcod_subcta"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Entre importes:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtImporte_ini">
        <property name="text">
         <string>0,00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Entre fechas:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dateFecha_inicio">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="btnBuscar">
        <property name="text">
         <string>Buscar</string>
        </property>
        <property name="icon">
         <iconset resource="../maya.qrc">
          <normaloff>:/Icons/PNG/search2.png</normaloff>:/Icons/PNG/search2.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>BUSQUEDA DE APUNTES CONTABLES.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dateFecha_inicio</tabstop>
  <tabstop>dateFecha_fin</tabstop>
  <tabstop>txtImporte_ini</tabstop>
  <tabstop>txtImporte_fin</tabstop>
  <tabstop>txtcod_subcta</tabstop>
  <tabstop>btnBuscar_subcta</tabstop>
  <tabstop>txtdesc_subcta</tabstop>
  <tabstop>btnBuscar</tabstop>
  <tabstop>tablaapuntes</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>btnAceptar</tabstop>
 </tabstops>
 <resources>
  <include location="../maya.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>frmBuscarApunteContable</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
